docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -22,6 +20,37 @@ The following APIs support {ccs}:
 * experimental:[] <<eql-search-api,EQL search>>
 * experimental:[] <<search-vector-tile-api,Vector tile search>>
 
+[discrete]
+[[ccs-prereqs]]
+=== Prerequisites
+
+* {ccs-cap} requires remote clusters. To set up remote clusters on {ess},
+see link:{cloud}/ec-enable-ccs.html[configure remote clusters on {ess}]. If you
+run {es} on your own hardware, see <<remote-clusters>>.
++
+To ensure your remote cluster configuration supports {ccs}, see
+<<ccs-supported-configurations>>.
```

Comment:
We're updating the section for this xref in https://github.com/elastic/elasticsearch/pull/80809.